setup.py

```diff
@@ -61,13 +61,12 @@
     keywords="task-scheduling parallel numpy pandas pydata",
     classifiers=[
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
```

Comment:
When do we add 3.9 here?

Reply:
Good question, I've not tried to run on 3.9 yet. We can open a PR which adds a Python 3.9 job to CI to see what issues pop up. So far, in `distributed`, I think https://github.com/dask/distributed/issues/4168 is the only reported 3.9 compatibility issue

Reply:
This was done in PR ( https://github.com/dask/distributed/pull/4460 )